website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -849,8 +853,8 @@ custom GitHub application for syncing secrets with GitHub repositories.
 
 ### Parameters
 
-- `name` `(string: <required>)` - Specifies a custom name for the GitHub
-application that is used when configuring the destination. Although this can be any name, we recommend using the same app name configured in GitHub.
+- `name` `(string: <required>)` - Specifies a custom name for the GitHub application that is used when configuring
+the destination. Although this can be any name, we recommend using the same app name configured in GitHub.
```

Comment:
```suggestion
- `name` `(string: <required>)` - Specifies a custom name for the GitHub
  application when configuring the sync destination. We recommend using the same 
  app name configured in GitHub.
```
Style correction: write in active voice

Also, why is using the same name considered best practice? Is it just so you can
tell at a glance which app the configuration relates to or is there another
reason?

Reply:
Yes, it's exactly that.